t/batches.t

```diff
@@ -170,9 +171,19 @@ subtest 'RPCAPI get_batch_job_result' => sub {
 
         is( $res->{nb_running}, @domains, 'correct number of runninng tests' );
         is( $res->{nb_finished}, 0, 'correct number of finished tests' );
+
+        my $res = $rpcapi->batch_status( { batch_id => $batch_id } );
```

Comment:
Avoid redeclaring `$res`. Preferably enclose each one in a separate scope, or use different names.